app/components/channel_list/header/index.ts

```diff
@@ -50,7 +40,7 @@ const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
         canCreateChannels,
         canJoinChannels,
         displayName: team.pipe(
-            switchMap((t) => of$(t.displayName)),
+            switchMap((t) => of$(t?.displayName)),
```

Comment:
should this be `of$(t?.displayName ||  '')` ?

Reply:
As of today, the component seems to handle beautifully the undefined case, so I see no reason to "default the value". Nevertheless, the component is not accepting undefined, but only string, but that will be solved when we fix "withObservables"'s types.